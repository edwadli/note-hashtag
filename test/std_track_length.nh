p1 = [1 5 6 7 1]
r1 = [1.0 2.0 1.0 0.5 2.0]
t1 = init track{chords = p1; durations = r1}
p2 = [4 1 2 3 4]
r2 = [1.0 2.0 3.0 4.0 5.0]
t2 = init track{chords = p2; durations = r2}
l1 = Length t1
l2 = Length t2
PrintEndline(StringOfFloat(l1))
PrintEndline(StringOfFloat(l2))