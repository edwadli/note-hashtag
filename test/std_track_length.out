6.500000
15.000000
